fa25-team-b/README.md

```
@@ -16,63 +16,174 @@
-From your command line:
+Continuing the work of students in the **Summer of 2025**, we are focusing on identifying **applicant movement patterns**, **portfolio biases**, and the **importance of age-restricted home offerings**.
 
-```bash
-# Clone this repository
-$ git clone [repo link]
+---
 
-# Further Instructions
-...
-```
+### The project is divided into three main base questions:
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)